src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -42,7 +43,7 @@ static int32_t g_config_specified_ciphersuites = 0;
 
 static void DetectCiphersuiteConfiguration()
 {
-    // This routine will always produce g_config_specified_ciphersuites = 0 on OpenSSL 1.0.x,
+    // This routine will always produce g_config_specified_ciphersuites = 1 on OpenSSL 1.0.x,
```

Comment:
If a non-portable build is built against OpenSSL 1.0.x it'll still return 0 with the current code.